bot/src/discord_commands/jibiki_all_search.js

```diff
@@ -0,0 +1,254 @@
+              }
+              name += sentence.tags.join(', ');
+              if (sentence.tags.length > 0) {
+                name += ') ';
+              }
+              name += sentence.sentence;
+
+              Object.values(sentence.translations).forEach((translation, i) => {
+                value += `${i + 1}. `;
+                if (translation.tags.length > 0) {
+                  value += '(';
+                }
+                value += translation.tags.join(', ');
+                if (translation.tags.length > 0) {
+                  value += ') ';
+                }
+                value += translation.sentence;
+              });
+
+              sentenceFields.push({
+                name,
+                value,
+              });
+            });
+
+            sentencePages.push({
+              embed: {
+                title: `Showing sentences for word ${entry.word.forms[0].kanji.literal !== null
+                  ? entry.word.forms[0].kanji.literal
+                  : entry.word.forms[0].reading.literal}`,
+                url: `https://jibiki.app?query=${encodeURIComponent(suffix)}`,
+                color: 16740862,
+                footer: {
+                  icon_url: 'https://jibiki.app/logo_circle.png',
+                  text: 'Powered by Jibiki',
+                },
+                author: {
+                  name: 'Powered by Jibiki',
+                  url: `https://jibiki.app?query=${encodeURIComponent(suffix)}`,
+                  icon_url: 'https://jibiki.app/logo_circle.png',
+                },
+                fields: sentenceFields,
+              },
+            });
+          }
+        }
+
+        wordPages.push({
+          embed: {
+            title: `Showing results for ${suffix}`,
+            description: `Page ${page + 1} out of ${Math.ceil(response.data.length / maxFieldsPerPage)} (${response.data.length} results)`,
+            url: `https://jibiki.app?query=${encodeURIComponent(suffix)}`,
+            color: 16740862,
+            footer: {
+              icon_url: 'https://jibiki.app/logo_circle.png',
+              text: 'Powered by Jibiki',
+            },
+            author: {
+              name: 'Powered by Jibiki',
+              url: `https://jibiki.app?query=${encodeURIComponent(suffix)}`,
+              icon_url: 'https://jibiki.app/logo_circle.png',
+            },
+            fields: wordFields,
+          },
+        });
+      }
+
+      return monochrome.getNavigationManager().show(
+        new Navigation(
+          msg.author.id,
+          true,
+          '🇯',
+          {
+            '🇯': NavigationChapter.fromContent(wordPages),
+            '🇰': NavigationChapter.fromContent(kanjiPages),
+            '🇪': NavigationChapter.fromContent(sentencePages),
+          },
+        ),
+        constants.NAVIGATION_EXPIRATION_TIME,
+        msg.channel,
+        msg,
+      );
+    } else if (response.status === 500) {
+      return msg.channel.createMessage({
```

Comment:
Please throw this as a FulfillmentError also. Otherwise this will appear in my logs as a successful command execution.

```js
const message = {
  embed: {
    title: 'Amy says no.',
    description: 'Jibiki; Internal server error.\nTry again later!',
    color: constants.EMBED_WRONG_COLOR,
  },
};

throw new FulfillmentError({
  publicMessage: message,
  logDescription: 'Jibiki 500 status',
});
```